config_template.cfg

```diff
@@ -192,6 +192,12 @@ FREESTREAM_VISCOSITY= 1.853E-5
 % Free-stream turbulence intensity
 FREESTREAM_TURBULENCEINTENSITY= 0.05
 %
+% Fix turbulence quantities to far-field values inside an upstream half-space
+TURB_FIXED_VALUES= NO
+%
+% Shift of the fixed values half-space, in space units in the direction of far-field velocity
+TURB_FIXED_VALUES_DOMAIN= -1.0
```

Comment:
I didnt understand this explanation in the config_template and with your PR explanation and 1 min on sheet of paper it was easy ... 
`The second parameter specifies the half-plane on which fixed values are applied. This half-plane consists of those points for which the dot product of their coordinate vector with the normalized far-field velocity vector is less than TURB_FIXED_VALUES_DOMAIN.`
Imo it is ok to be verbose in the cfg_template 
(Now that I got it, this short sentence makes sense as well... 🙈 i think the space units 🚀 👨‍🚀 threw me off)

Reply:
Yeah maybe "spatial units" or "length units" are better terms